ddht/v5_1/app.py

```diff
@@ -102,13 +93,19 @@ async def run(self) -> None:
         )
         network = Network(client=client, bootnodes=bootnodes,)
 
-        logger.info("Protocol-Version: %s", self._boot_info.protocol_version.value)
-        logger.info("DDHT base dir: %s", self._boot_info.base_dir)
-        logger.info("Starting discovery service...")
-        logger.info("Listening on %s:%d", listen_on, port)
-        logger.info("Local Node ID: %s", encode_hex(enr_manager.enr.node_id))
-        logger.info(
+        if self._boot_info.is_rpc_enabled:
+            handlers = get_core_rpc_handlers(network.routing_table)
```

Comment:
This is missing the `enr` parameter.

Reply:
Looks like the `to_dict` decorator isn't preserving the function arguments... :cry: